src/TimeSync.cpp

```diff
@@ -151,8 +151,11 @@ namespace time_sync {
                 return false;
```

Comment:
Nit: Maybe update the headerfile to mention that `createPerformanceLogger` function creates logger for each process